pydis_site/apps/api/models/bot/nomination.py

```diff
@@ -42,6 +31,10 @@ class Nomination(ModelReprMixin, models.Model):
         help_text="When the nomination was ended.",
         null=True
     )
+    reviewed = models.BooleanField(
+        default=False,
+        help_text="Whether voting message have been made."
```

Comment:
Another small correction
```suggestion
        help_text="Whether a review was made."
```